Gruntfile.js

```diff
@@ -899,6 +899,26 @@ module.exports = function( grunt ) {
         },
     );
 
+    grunt.registerTask(
+        'testNotifications',
+        'Runs registered notification mechanisms with test messages.',
+        function() {
+            var privateParams = grunt.config.get( 'privateParams' ),
+                slackCreds = privateParams.slack_creds || {};
+            if (
+                slackCreds.authToken &&
+                slackCreds.botToken &&
+                slackCreds.channels
+            ) {
+                grunt.config.set(
+                    'slackNotificationMessage',
+                    { text: 'Testing Slack Notification Messages' }
+                );
+                grunt.task.run( 'slack_api:notify_build' );
+            }
+        },
```

Comment:
that trailing comma is ok on Javascript function calls eh? That's news to me!

Reply:
afaik it's optional in most places like this where there are not any additional parameters, but many linters want you to add it anyways. I think the idea is that it leads to cleaner diffs in the future if you add new parameters or reorder the existing ones because you don't have to much with commas anywhere.